src/app/frontend/replicasetdetail/replicasetdetail_controller.js

```diff
@@ -156,6 +165,19 @@ export default class ReplicaSetDetailController {
    */
 
   /**
+   * Updates replicas count in replica set
+   * @param {number} replicasCount
+   * @param {function(!backendApi.ReplicaSetDetail)=} opt_callback
+   * @param {function(!angular.$http.Response)=} opt_errback
+   * @private
+   */
+  updateReplicas_(replicasCount, opt_callback, opt_errback) {
+    this.replicaSetDetail.podsDesired = replicasCount;
```

Comment:
The open question what REST endpoint should this have? I leave this to you for investigation. Think of `/repicasets/:namespace/:name/pods/update/` or `/repicasets/pods/:namespace/:name/update/`